PositiveAirwayPressureDevices/R4/resources/Questionnaire-R4-PositiveAirwayPressureDevicesFaceToFace.json

```diff
@@ -29,339 +29,90 @@
-                "language": "text/cql",
-                "expression": "\"PositiveAirwayPressureDevicePrepopulation\".Today"
-              }
-            }
-          ]
-        },
-        {
-          "linkId": "3.2",
-          "text": "Is this a F2F evaluation of the patient’s need for a PAP device for OSA? ",
-          "type": "boolean",
           "extension": [
             {
               "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
               "valueExpression": {
                 "language": "text/cql",
-                "expression": "\"PositiveAirwayPressureDevicePrepopulation\".True"
+                "expression": "\"PositiveAirwayPressureDevicePrepopulation\".RelevantDiagnoses"
               }
             }
-          ]
+          ],
+          "linkId": "4.1",
+          "type": "open-choice",
+          "text": "Relevant Diagnoses",
+          "answerOption": [],
+          "repeats": "true"
         },
         {
-          "linkId": "3.3",
-          "text": "Order purpose: ",
+          "linkId": "4.2",
+          "text": "Other diagnoses",
           "type": "group",
-          "enableWhen": [
-            {
-              "question": "3.2",
-              "operator": "=",
-              "answerBoolean": true
-            }
-          ],
           "item": [
             {
-              "linkId": "3.3.1",
-              "type": "choice",
-              "required": false,
-              "answerOption": [
-                {
-                  "valueCoding": {
-                    "code": "Initial Evaluation"
-                  }
-                },
-                {
-                  "valueCoding": {
-                    "code": "Re-evaluation"
-                  }
-                }
-              ]
-            },
-            {
-              "linkId": "3.3.2",
-              "text": "Is there evidence of continued use of the PAP/CPAP device and supplies?",
-              "type": "boolean",
               "extension": [
                 {
                   "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
                   "valueExpression": {
                     "language": "text/cql",
-                    "expression": "\"PositiveAirwayPressureDevicePrepopulation\".True"
+                    "expression": "\"PositiveAirwayPressureDevicePrepopulation\".OtherDiagnoses"
                   }
                 }
               ],
-              "enableWhen": [
-                {
-                  "question": "3.3.1",
-                  "operator": "=",
-                  "answerCoding": {
-                    "code": "Re-evaluation"
-                  }
-                }
-              ]
+              "linkId": "4.2.1",
+              "text": "Other",
+              "type": "open-choice",
+              "answerOption": [],
+              "repeats": "true"
             },
             {
-              "linkId": "3.3.3",
+              "linkId": "4.2.2",
```

Comment:
We don't need description text here. Open-choice itself allows free text

Reply:
If typing and entering the choice, it seems like the description is one of the diagnoses. That is why I explicitly add a text box for better UX experience.